src/main/java/uk/gov/ons/ssdc/supporttool/rasrm/client/RasRmSampleServiceClient.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.ons.ssdc.supporttool.rasrm.client;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import uk.gov.ons.ssdc.supporttool.rasrm.model.dto.rest.RasRmSampleSummaryDTO;
+import uk.gov.ons.ssdc.supporttool.rasrm.model.dto.rest.RasRmSampleSummaryResponseDTO;
+
+@Component
+public class RasRmSampleServiceClient {
+
+  @Value("${ras-rm-sample-service.connection.scheme}")
+  private String scheme;
+
+  @Value("${ras-rm-sample-service.connection.host}")
+  private String host;
+
+  @Value("${ras-rm-sample-service.connection.port}")
+  private String port;
+
+  public RasRmSampleSummaryResponseDTO createSampleSummary(
+      int totalSampleUnits, int expectedCollectionInstruments) {
+    RasRmSampleSummaryDTO sampleSummaryDTO = new RasRmSampleSummaryDTO();
+    sampleSummaryDTO.setTotalSampleUnits(totalSampleUnits);
+    sampleSummaryDTO.setExpectedCollectionInstruments(expectedCollectionInstruments);
+
+    RestTemplate restTemplate = new RestTemplate();
+    UriComponents uriComponents = createUriComponents("samples/samplesummary");
+
+    return restTemplate.postForObject(
```

Comment:
what will happen if this post fails?

Reply:
it will throw a HttpClientException, which will cause the caller to fail... ideally meaning that everything gets rolled back and is retried indefinitely